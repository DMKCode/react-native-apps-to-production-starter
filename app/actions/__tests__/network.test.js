import { changeNetworkStatus } from '../network';

it('creates a CHANGE_CONNECTION_STATUS action', () => {
  expect(changeNetworkStatus('offline')).toMatchSnapshot();
});
