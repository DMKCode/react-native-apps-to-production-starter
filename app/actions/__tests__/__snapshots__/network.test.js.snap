// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates a CHANGE_CONNECTION_STATUS action 1`] = `
Object {
  "status": "offline",
  "type": "CHANGE_CONNECTION_STATUS",
}
`;
